src/client.ts

```diff
@@ -310,6 +312,10 @@ export default class Client implements ClientInterface {
       vscode.commands.registerCommand(
         Command.Update,
         this.updateServer.bind(this)
+      ),
+      vscode.commands.registerCommand(
+        Command.OpenGemHomepage,
+        this.openGemHomepage.bind(this)
```

Comment:
NIT: let's rename both the command and function to be `openLink` so that it's more generic. We may want to link to other places in the editor in the future and then we can reuse this function.